src/Engine/OverlayCache.java

```diff
@@ -59,6 +59,9 @@ public ArrayList<GameOverlay> getNormalOverlays(final OverlayMode mode, final Ma
       ArrayList<Unit> threats = new ArrayList<>();
 
       final Army viewer = drawableMap.viewer;
+      if( null == viewer )
+        return new ArrayList<>();
```

Comment:
I guess I should ask the usual question about asking for a perspective for the null viewer. Do we do that?

Reply:
Yes. It's in the commit message: multi-human+AI fog games.
The map is fully hidden during AI turns there, I believe... though maybe it should account for "all the humans are allied".